### DISCUSSION TITLE
Ionosphere - how to write the VTEC map using the ionosphereMap parametrization

### DISCUSSION THREAD

**Discussion by laplace76** [groops-devs/groops#255] [Q&A] [answered]
I'd like to know if it is possible to extract the VTEC estimated in the orbit determination example and write it on a file,  as the ionosphereMap parametrization does not have any variable defining the output file.

**Comment by tmayerguerr** [comment]
Hi,

The option to read/write VTEC maps was missing for a long time, but is now included in the main branch. Since the estimation is based on a one-layer model, I don't know if it also works for LEO satellites.

Best regards
Torsten

**Comment by laplace76** [comment]
yes, this is understood. The thing is that the variables selectReceivers, outputfileGriddedTimeSeries, outputGrid, outputTimeseries and inputfileGriddedDataTimeSeries do not show up in the gui. I think this is because I am using some configuration files I have created using an older version of GROOPS (one week ago I have dovnloaded and rebuilt the updated version, including changes for the iono): Is there any way to update the old config files (adding the new variables) automatically?

**Comment by tmayerguerr** [comment, marked as answer]
The file `groops.xsd` contains all the options and is read by the GUI. This file must be consistent with the groops version. The GUI itself does not need to be updated. Open old groops configuration files with the GUI using the current `groops.xsd` should not cause any problems and the new options should be available.

**Reply by laplace76** [reply]
It worked, thank you.
